packages/edit-post/src/classic.scss

```diff
@@ -1,12 +1,14 @@
-// This needs specificity to override the editor styles.
+// Provide baseline auto margin for centering blocks.
+// Specificity is kept at this level as many classic themes output
+// rules like figure { margin: 0; } which would break centering.
 .editor-styles-wrapper .wp-block {
 	margin-left: auto;
 	margin-right: auto;
 }
 
-// Depreacted style needed for the block widths and alignments.
-// for themes that don't support the new layout (theme.json)
-.wp-block {
+// Deprecated style needed for the block widths and alignments.
+// for themes that don't support the new layout (theme.json).
+html :where(.wp-block) {
 	max-width: $content-width;
 
 	// Provide every block with a default base margin. This margin provides a consistent spacing
```

Comment:
Noting that these margins can also get removed by `figure { margin: 0; }`

Reply:
I'd personally prefer to keep them there, as they as I recall were the most disruptive to work with, and the reduced specificity of those margins were the biggest win of this approach. It should also be less disruptive as it's vertical margins.

However I'm happy to move them up either in this PR or as a followup if it turns out to be necessary?

Reply:
>  the biggest win of this approach.

I'm curious, in which situation the smaller specificity benefit classic themes here. What kind of styles classic themes apply here?

Reply:
I'm thinking mostly of developing new classic themes, or refactoring older ones.

Reply:
I agree with @jasmussen that it should be fine to keep these where they are. I tested in a few themes and didn't notice this causing any issues. If they did interfere with that `figure` rule, it'll just be the vertical margin (and may actually have no effect if there's margin on the items above + below the `figure` anyway).

Reply:
Thanks for looking Kjell. Cool, in any case it's an easy follow up which I will be keeping my eyes peeled on and ready to change if need be. I'd like to land this PR tomorrow or Monday, so there's still time to chime in here.